bcgov/moh-prime
# PRIME-2053 do not send renewal emails to absent enrollees

[PRIME-2053](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-2053)

This is to be merged after [PR 1884](https://github.com/bcgov/moh-prime/pull/1884)

Pre test:

1. Add a new migration.
3. In `management/enrollees/renewal` endpoint add [AllowAnnonymous] and comment out out the authorize role (found on line 45 of the EmailsController.cs)

How to test:

1. Create 2 enrollees and approve them
2. Add an absence to one of the enrollees
3. Hit this endpoint from postman http://localhost:5000/api/Emails/management/enrollees/renewal
4. Check your mailhog (only one enrollee should get an email)
